client/kafka/kafka.go

```diff
@@ -70,6 +71,11 @@ func NewMessage(t string, b interface{}) *Message {
 	return &Message{topic: t, body: b}
 }
 
+// SetHeader allows to set or override a message header.
+func (m *Message) SetHeader(key, value string) {
```

Comment:
Maybe just a test to confirm that is is set correctly...

Reply:
I can add an integration test that verifies that header is received as it was sent; do you think it would be useful?

Reply:
a simple unit test should be enough